src/MenuControlSystem.cpp

```diff
@@ -0,0 +1,36 @@
+#include "MenuControlSystem.h"
+
+void MenuControlSystem::on_tick()
+{
+	const auto& key = KeyState::get();
+
+	for (auto&& [entity, menu, key_binding] : access_storage().each())
+	{
+		if (key.is_pressed(key_binding.escape))
+		{
+			menu.m_fsm.trigger(MenuTransition::EXIT);
+			Engine::get_instance().quit();
+		}
+		if (key.is_pressed(key_binding.a))
```

Comment:
A is a bit of a weird choice for pause keybinding, especially since somebody might assume AWSD controls. Maybe make you could use P for both pause and unpause, depending on which state you're in currently?

Reply:
Pause functionality is not in use, which is why we decided not to change it.